src/view/com/lists/ProfileLists.tsx

```diff
@@ -60,27 +60,26 @@ export const ProfileLists = React.forwardRef<SectionRef, ProfileListsProps>(
       error,
       refetch,
     } = useProfileListsQuery(did, opts)
-    const isEmpty = !isFetching && !data?.pages[0]?.lists.length
-
     const items = React.useMemo(() => {
+      const hasListData = data?.pages[0]?.lists.length
       let items: any[] = []
-      if (isError && isEmpty) {
+      if (isError && !isFetching && !hasListData) {
         items = items.concat([ERROR_ITEM])
       }
       if (!isFetched && isFetching) {
         items = items.concat([LOADING])
-      } else if (isEmpty) {
+      } else if (isFetched && !hasListData) {
         items = items.concat([EMPTY])
       } else if (data?.pages) {
         for (const page of data?.pages) {
           items = items.concat(page.lists)
         }
       }
-      if (isError && !isEmpty) {
+      if (isError && !isFetching && hasListData) {
```

Comment:
The way this original logic was written, `LOAD_MORE_ERROR_ITEM` would be added whenever `isError` + `isFetching` were both true 

That behavior did not seem intentional to me as it has the potential to cause some weird bugs (for example if `isError` remains true while refetching from `ErrorMessage`, then `LoadMoreRetryBtn` will flash during the API call)

Can revert the change if I'm missing something